introduction.md

```diff
@@ -1,2 +1,6 @@
 Hi, I'm Gosia Teresa. I love rasperries and bears.
-My favourite Code School path is definitely Ruby!
\ No newline at end of file
+My favourite Code School path is definitely Ruby!
+My deadly skills include:
+*Git
+*Ruby
+*Pillow killing
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?